pkg/manage/cmd_set_password.go

```diff
@@ -25,63 +26,84 @@ This command sets the password of a user by a given user id.
 
 // CmdSetPassword initializes the set-password command.
 func CmdSetPassword(cfg *ClientConfig) *cobra.Command {
-	var userID int64
-	var password string
-
 	cmd := &cobra.Command{
 		Use:   "set-password",
 		Short: "Sets an user password.",
 		Long:  setPasswordHelp,
-		RunE: func(cmd *cobra.Command, args []string) error {
-			ctx, cancel := context.WithTimeout(context.Background(), cfg.Timeout)
-			defer cancel()
+	}
 
-			service, close, err := Dial(ctx, cfg.Address)
-			if err != nil {
-				return fmt.Errorf("connecting to gRPC server: %w", err)
-			}
-			defer close()
+	userID := cmd.Flags().Int64P("user_id", "u", 1, "ID of the user account.")
+	password := cmd.Flags().StringP("password", "p", "admin", "New password for the user.")
+	onlyUpdate := cmd.Flags().Bool("update", false, "Do not overwrite an existing password.")
 
-			req := &proto.SetPasswordRequest{
-				UserID:   userID,
-				Password: password,
-			}
+	cmd.RunE = func(cmd *cobra.Command, args []string) error {
+		ctx, cancel := context.WithTimeout(context.Background(), cfg.Timeout)
+		defer cancel()
+
+		service, close, err := Dial(ctx, cfg.Address)
+		if err != nil {
+			return fmt.Errorf("connecting to gRPC server: %w", err)
+		}
+		defer close()
+
+		req := &proto.SetPasswordRequest{
+			UserID:     *userID,
+			Password:   *password,
+			OnlyUpdate: *onlyUpdate,
+		}
 
-			if _, err := service.SetPassword(ctx, req); err != nil {
-				return fmt.Errorf("reset password: %w", err)
+		resp, err := service.SetPassword(ctx, req)
+		if err != nil {
+			return fmt.Errorf("reset password: %w", err)
+		}
+
+		if *onlyUpdate {
+			msg := "Password did already exist."
```

Comment:
Proposal: `msg := "User does already have an existing password. Password was not changed."`

Reply:
Done